wezterm-gui/build.rs

```diff
@@ -168,7 +168,10 @@ END
             .join("WezTerm.app")
             .join("Contents")
             .join("Info.plist");
-        let dest_plist = repo_dir.join("target").join(profile).join("Info.plist");
+        let build_target_dir = std::env::var("CARGO_BUILD_TARGET_DIR")
```

Comment:
Should this be `CARGO_TARGET_DIR`? When is `CARGO_BUILD_TARGET_DIR` set?

https://doc.rust-lang.org/cargo/appendix/glossary.html?highlight=CARGO_TARGET_DIR#target